examples/4_train_policy_with_script.md

```diff
@@ -0,0 +1,157 @@
+This tutorial will explain the training script, how to use it, and particularly the use of Hydra to configure everything needed for the training run.
```

Comment:
This in-depth tutorial is important, but it would be nice to have a specific markdown file called: `5_reproduce_sota.md` with a structured list of training commands + pointers to our model page on the hub + evaluation commands of the pretrained models.

```
# Table of results

TODO Table with link to model page on the hub + link to section in the markdown file

## Diffusion policy on Pusht

Model card on the hub:

Training command:
```
TODO
```

Eval Command
```
TODO
```

## Aloha

...


```

Reply:
I don't think this is a good idea. It adds more not-maintainable documentation. These details live with the model cards and I don't think they should be repeated elsewhere. IMO, we should just have a zoo.md style file that points to the model cards.

Reply:
What do you think of adding a `5_reproduce_sota.md` which would be a tutorial on how to access the model card list, and how from the model card, access the training command?

Maybe long term with can avoid this maintainable documentation, but right now I have no idea how to access these training commands.

Reply:
How about I put the commands on the model cards. Then you have command, configuration, and commit hash all in one place. Then I can add a section in the main README to talk about the model cards.

Reply:
Added to this PR.

On your thought: I suppose... it's a one-liner right now, so a script doesn't add much value right?

Reply:
Ah for the README. Yeah that's a good idea :)